eq-author/src/App/metadata/MetadataPage.test.js

```diff
@@ -74,4 +80,21 @@ describe("MetadataModal", () => {
     });
     expect(renderContent).toMatchSnapshot();
   });
+
+  it("should render no metadata message when no metadata", () => {
+    contentProps = {
+      ...contentProps,
+      data: {
+        questionnaire: {
+          id: questionnaireId,
+          metadata: [],
+        },
+      },
+    };
+    const instance = wrapper.instance();
+    const renderContent = instance.renderContent({
+      ...contentProps,
+    });
+    expect(renderContent).toMatchSnapshot();
```

Comment:
Can we look for the message here, instead of the snapshot?

Reply:
I've been unable to find a way to do this. Due to the way the metadata component was constructed, we need to call the renderContent() method on the instance to get the content. I don't know a way to do a find from the results of the renderContent(). Using a snapshot follows the way the rest of the tests work